pages/linux/tcptraceroute.md

```diff
@@ -15,11 +15,11 @@
 
 `tcptraceroute {{host}} -p {{source_port}} -s {{source_address}}`
 
-- Set the initial and maximum TTL:
+- Set the first and maximum TTL:
```

Comment:
I would keep initial here.

Reply:
Refer to https://github.com/tldr-pages/tldr/pull/3849#discussion_r382273547

Reply:
Okay then. My thoughts were that "initial" is more clear in tech documentation.